### DISCUSSION TITLE
Adding New Tag to override views

### DISCUSSION THREAD

**Discussion by cdigruttola** [shopperlabs/framework#236] [Ideas]
Is it possible to add a new tag that can be used as follow, in order to override, when needed, the views?
 php artisan vendor:publish --tag=shopper-views

**Comment by mckenziearts** [comment]
Hey @cdigruttola, good news, this is already supported:

```bash
php artisan vendor:publish --tag=shopper-views
```

You can also publish `shopper-config`, `shopper-lang`, and `shopper-migrations` the same way. Let us know if you hit a specific view that isn't covered.
